backend/rdp/output.c

```diff
@@ -107,6 +107,9 @@ static bool rfx_swap_buffers(
 	int nrects;
 	pixman_box32_t *rects =
 		pixman_region32_rectangles(damage, &nrects);
+	if (nrects == 0) {
```

Comment:
Just in case, could you move the check at the beginning of the function, and use `pixman_region32_not_empty` to check whether the region is empty?

Reply:
Thanks, that's a good suggestion! I've implemented it.